Cleaning_PSGC.ipynb

```diff
@@ -572,7 +572,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We see alternate names again but notice the `(Pob.)` suffixes. A quick Google search shows that it's short for `Poblacion` which is used to denote the commercial and industrial center of a city. Let's just strip these."
+    "We see alternate names again but notice the `(Pob.)` suffixes. A quick Google search shows that it's short for `Poblacion` which is used to denote the commercial and industrial center of a city. Let's stash those and add them as aliases"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "barangays_pob = barangays[barangays.location.str.contains('\\(Pob.\\)')].copy()"
```

Comment:
From my understanding, `Adams (Pob.)` should end up having two rows in the reference file:
`Adams (Pob.)`
`Adams`

One with the original `(Pob.)` suffix and another with that suffix stripped.  Are we still stripping the suffix at any point? It doesn't look that way here:

`"barangays_pob = barangays[barangays.location.str.contains('\\(Pob.\\)')].copy()"`

Reply:
That part of the notebook is where we stash those with `(Pob.)` to be added later. The rest of the cleanup that strips that suffix are still part of the code.

Reply:
Ahh got it, it just didn't show up here in changes because it wasn't changed. Haha. Duh. Okay thanks :)